OpenOversight/tests/routes/test_other.py

```diff
@@ -54,3 +54,16 @@ def test_user_can_access_profile_differently_cased(mockdata, client, session):
 
         # Should use username in db
         assert "TEST_USER" not in rv.data.decode(ENCODING_UTF_8)
+
+
+def test_timezone_setting(client):
```

Comment:
fixed the test. The important thing is that session as a test fixture (from adding it as a parameter like `client`) refers to the database session. Here we want the client browser's session

Reply:
Ahhhh, that makes sense. I presumed it was the applications client.